XenModel/Messages.resx

```diff
@@ -7429,6 +7429,9 @@ This will permanently delete and reinitialize all local storage on the servers.
   <data name="MANAGEMENT_NETWORK_WITH_POOL" xml:space="preserve">
     <value>{0} (management) on {1}</value>
   </data>
+  <data name="MANAGE_UPDATES_PAGE_AUTOMATED_UPDATES_NOT_AVAILABLE" xml:space="preserve">
+    <value>Automated updates are not available</value>

```

Comment:
Good point @kc284 

Although I would shorten it a bit to:
"Automated updates are not supported for these servers"
what is the plural form of POOL_FORBIDS_AUTOMATED_UPDATES="Automated updates are not supported for this server" that is already being used.

I suggest we should just show that one if there is only one server in the list and also add the plural version that would be shown when there are more than one servers shown.